src/domain/configure_ftp_cameras.py

```diff
@@ -161,7 +161,7 @@ def accept_and_close(self):
                               camera.type == Camera.CameraTypes.FTPCamera)
             for camera in orphan_cameras:
                 cameras.remove(camera)
-            for camera in cameras:
+            for camera in cameras[:]:
```

Comment:
for camera in tuple(cameras):

Reply:
Ok, but what's the diff with the above? for my education.. :)

Reply:
With the [:] you're creating a list, which is less efficient and consumes more memory compared to a tuple. Lists are mutable objects whereas tuples are immutable, and in this case you don't need to add any element on the list, because you're just iterating it one at the time, only for picking its elements. A tuples is better suited for this (well, an iterator would be better, but it cannot be used, because you're modifying in-place the original list while iterating it).

Reply:
Thanks a lot!